CONTRIBUTING.md

```diff
@@ -158,3 +223,321 @@ including how to setup a local Kubernetes cluster, can be found in
+or frequently failing test.
+
+To create a failing test issue or PR:
+
+* label your issue or PR with `kind/failing-test`
+
+TODO: As more automation is used in the continuous integration pipeline, we will
+be able to automatically generate an issue for failing tests.
+
+#### Support
+
+A `kind/support` label categorizes issue as related to a support request.
+
+To create a support issue or PR:
+
+* label your issue or PR with `kind/support`
+* title the issue or PR with a short description of your support request
+* answer all of the questions in the support issue template
+
+_Note this was previously called `support`. We will add the `kind/support` label
+to all issues currently labeled with `support`.
+
+### Area Assignment
+
+Area labels begin with `area/` and identify areas of interest or functionality
+to which an issue relates. An issue or PR could have multiple areas. These labels are
+used to sort issues and PRs into categories such as:
+
+* operating systems
+* cloud platform, 
+* functional area,
+* operating or legal area (i.e., licensing), 
+* etc.
+
+A list of areas is maintained in the [`docs/labels.md`](docs/labels.md) document.
+
+An area may be changed, added or deleted during issue or PR triage.
+
+### Size
+
+Size labels beting with `size/` and estimate the relative complexity or work
+required to resolve an issue or PR. 
+
+TODO: For submitted PRs, the size can be automatically calculated and the
+appropriate label assigned.
+
+Size labels are specified according to lines of code; however, some issues may
+not relate to lines of code submission such as documentation. In those cases,
+use the labels to assign an equivalent complexity or size to the task at hand.
+
+Size labels include:
+
+* `size/XS` -- denotes a extra small issue, or PR that changes 0-9 lines, ignoring generated files
+* `size/S` -- denotes a small issue, or PR that changes 10-29 lines, ignoring generated files
+* `size/M` -- denotes a medium issue, or PR that changes 30-99 lines, ignoring generated files
+* `size/L` -- denotes a large issue, or PR that changes 100-499 lines, ignoring generated files
+* `size/XL` -- denotes a very large issue, or PR that changes 500+ lines, ignoring generated files
+
+Size labels are defined in the [`docs/labels.md`](docs/labels.md) document.
+
+### Triage
+
+As soon as new issue are submitted, they must be triaged until they are ready to
+work. The maintainers may assign the following labels during the issue triage
+process:
+
+* `triage/duplicate`  -- indicates an issue is a duplicate of other open issue
+* `triage/needs-information` -- indicates an issue needs more information in order to work on it
+* `triage/not-reproducible` -- indicates an issue can not be reproduced as described
+* `triage/unresolved` -- indicates an issue that can not or will not be resolved
+
+Triage labels are defined in the [`docs/labels.md`](docs/labels.md) document.
```

Comment:
need to update doc/labels.md -> doc/github-labels.md everywhere

Reply:
Done.